packages/contentstack-import/src/import/modules-js/entries.js

```diff
@@ -486,7 +498,17 @@ module.exports = class ImportEntries {
           return resolve();
         })
         .catch((error) => {
-          addlogs(this.config, chalk.red("Failed to create entries in '" + lang + "' language"), 'error');
+          addlogs(
+            this.config,
+            chalk.red(
+              "Failed to create entries in '" +
+                lang +
+                "' language. " +
+                'Title of the failed entry: ' +
+                JSON.parse(error.request.data).entry.title,
```

Comment:
@cs-raj Could you please check do we need  **JSON.parse(error.request.data).entry.title here** ?

Reply:
@aman19K  changes are done and removed the redundant code